### PR TITLE
PR: Update to Phoenix 1.5.3 and LiveView 0.13.2 issue #17

### PR DESCRIPTION
**Pull request by nelsonic**
As the title of the PR suggests this Pull Requests contains: 
+ [x] Update to the latest version of Phoenix & LiveView #17 ⬆️ 
+ [x] Updates to the `README.md` to reflect new (simplified) setup process. 📝

### CODE REVIEW

**Comment by nelsonic** [comment]
Build failing due to diverged dependencies:

```
Dependencies have diverged:
* plug (Hex package)
  the dependency plug 1.9.0
  > In deps/plug_cowboy/mix.exs:
    {:plug, "~> 1.7", [env: :prod, hex: "plug", repo: "hexpm", optional: false]}
  does not match the requirement specified
  > In deps/phoenix/mix.exs:
    {:plug, "~> 1.10", [env: :prod, hex: "plug", repo: "hexpm", optional: false]}
  Ensure they match or specify one of the above in your deps and set "override: true"
* plug_cowboy (Hex package)
  the dependency plug_cowboy 2.1.2
  > In mix.exs:
    {:plug_cowboy, "~> 2.0", [env: :prod, repo: "hexpm", hex: "plug_cowboy"]}
  does not match the requirement specified
  > In deps/phoenix/mix.exs:
    {:plug_cowboy, "~> 1.0 or ~> 2.2", 
  [env: :prod, hex: "plug_cowboy", repo: "hexpm", optional: true]}
  Ensure they match or specify one of the above in your deps and set "override: true"
** (Mix) Can't continue due to errors on dependencies
The command "mix deps.get" failed and exited with 1 during .
Your build has been stopped.
```

https://travis-ci.com/github/dwyl/phoenix-liveview-counter-tutorial/builds/168954882#L263
image

**Comment by nelsonic** [comment]
This PR is only big because I recreated the App from scratch so there's a bunch of Phoenix boilerplate and `package-lock.json` 🙄 
In actual fact the `README.md` was _significantly_ reduced! That's a _great_ thing because the tutorial is streamlined and more beginner friendly. 🎉

File: README.md [hunk 1]
```diff
@@ -93,11 +106,12 @@ elixir -v
 You should expect to see output similar to the following:
 
 ```elixir
-Elixir 1.10.2 (compiled with Erlang/OTP 22)
+Elixir 1.10.3 (compiled with Erlang/OTP 22)
 ```
 This informs us we are using `Elixir version 1.10.2`
```
Comment (SimonLab) [inline README.md:111]:
```suggestion
This informs us we are using `Elixir version 1.10.3`
```

File: README.md [hunk 2]
```diff
@@ -922,13 +564,13 @@ and updated data (_in our case the `:val` count_)
...
+> 🏁 At the end of Step 4 you should have a file similar to:
+[`lib/live_view_counter_web/live/counter.ex`](https://github.com/dwyl/phoenix-liveview-counter-tutorial/blob/7e75ba0cfd7f170dc022cfdf62af380d70cc1496/lib/live_view_counter_web/live/counter.ex)
 
 <br />
 
 
-### Step 12: Create the `live` route in `router.ex` and define the root layout
+### Step 5: Create the `live` Route in `router.ex`
 
 Now that we have created our Live handler function in Step 11,
```
Comment (SimonLab) [inline README.md:575]:
```suggestion
Now that we have created our Live handler function in Step 4,
```

**Comment by nelsonic** [comment]
@SimonLab thanks for fixing those two issues. 👍

**Comment by nelsonic** [comment]
@SimonLab thanks! 🙌

**Pull request merged**